mmv1/third_party/terraform/services/identityplatform/resource_identity_platform_config_test.go

```diff
@@ -0,0 +1,112 @@
+package identityplatform_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccIdentityPlatformConfig_update(t *testing.T) {
+	acctest.SkipIfVcr(t)
```

Comment:
Looks like the test is skipped in VCR (presubmit tests). Are you able to run the test and get it passed locally?

Reply:
I'm commenting out the skip VCP switch. My local env is broken for some other reasons, so I prefer to run it in github. Note that VCR is skipped in the first place because of: https://github.com/hashicorp/terraform-provider-google/issues/13327

https://github.com/hashicorp/terraform-provider-google/issues/13327

Reply:
Thanks for the context.